CHANGELOG.md

```diff
@@ -1,9 +1,25 @@
 # Changelog
 
-## Release v0.9.8
+## Release v0.9.9
 
 ### Added
+
 - Support for setting TTL on resources and configuring resource heartbeating
```

Comment:
Maybe remove this line if this is covered by L9?

Reply:
Done.